### PR TITLE
Adding uwp and .netcore local feed package installer 

### PR DESCRIPTION
**Pull request by jinujoseph**
@srivatsn

### CODE REVIEW

File: LocalFeed/src/netfx_NETCoreSDK/IlcComponents.xslt [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:wix="http://schemas.microsoft.com/wix/2006/wi"
+  xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension">
+
+  <xsl:output method="xml" encoding="UTF-8" indent="yes"/>
+  
+  <xsl:param name="blacklist" select="document('blacklist.xml')/BlackList/Assembly"/>
```
Comment (nguerrera) [inline LocalFeed/src/netfx_NETCoreSDK/IlcComponents.xslt]: This term should be avoided. Replace with ExcludeList or BlockList or something.

**Comment by nguerrera** [comment]
Long/medium term, I am wondering if these really belong in this repository. Already, the templates directory is quite different from the rest of the repo and it's causing engineering headaches as I try to clean up the build process. I think we should factor out these components that have a different, Windows-only build process and are not used to actually develop and test Microsoft.NET.Sdk.

**Comment by livarcocc** [comment]
I agree with @nguerrera. Let's get together and find a better home for these assets in a separate/new repo.

File: LocalFeed/src/netfx_NETCoreSDK/IlcComponents.xslt (hunk 1 above)
Comment (jinujoseph) [inline LocalFeed/src/netfx_NETCoreSDK/IlcComponents.xslt]: ah , agree will fix that

**Comment by jinujoseph** [comment]
ya , Sri and myself debated creating a new repro or does it fit in sdk discussion, let's chat about it .

**Comment by srivatsn** [comment]
Ya given where SDK has evolved I agree it's not the right place. Let's just create a new repo for the package feed installer. @davkean or @nguerrera - can one of you do that?

**Comment by davkean** [comment]
I can, I also agree we should pull the templates out of SDK. The thinking around templates was there was going to be unification with the CLI templates but that never happened.

**Comment by davkean** [comment]
Maybe we should pull templates back into project system.

**Comment by jinujoseph** [comment]
Closing this, This Installer has been merged to a different repo. Thanks @davkean for creating a new repo.

**Comment by nguerrera** [comment]
> Maybe we should pull templates back into project system.

I think that makes sense.

**Pull request closed without merging**